src/test/java/core/model/TranslationFacilityTest.java

```diff
@@ -0,0 +1,75 @@
+package core.model;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.MissingResourceException;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+class TranslationFacilityTest {
+
+    @Test
+    void getTranslator() {
+        // given
+        TranslationFacility.setTranslator(null);
+
+        // when
+        final var translator = TranslationFacility.getTranslator();
+
+        // then
+        assertThat(translator.getLanguage()).isEqualTo(Translator.LANGUAGE_NO_TRANSLATION);
+    }
+
+    @Test
+    void setLanguage_with_German() {
+        // given
+        TranslationFacility.setLanguage("German");
+
+        // when
+        final var translator = TranslationFacility.getTranslator();
+
+        // then
+        assertThat(translator.getLanguage()).isEqualTo("German");
+    }
+
+    @Test
+    void setLanguage_unknown_throws_exception() {
+        // when-then
+        assertThatThrownBy(() -> TranslationFacility.setLanguage("foobar")).isInstanceOf(MissingResourceException.class);
+    }
+
+    @Test
+    void setTranslator() {
```

Comment:
Do we really need this test, given `getTranslator` pretty much does the same thing?

Reply:
> Do we really need this test, given `getTranslator` pretty much does the same thing?

Yes, because one test is explicit for `getTranslator` and explicit for `setTranslator`.
For `setTranslator` is important that you cannot set it to `null`.
It will still return a translator which is the initial translator with no translation.

The test for `setTranslator` can be different.
I changed it.

Review Finding 2